azkaban-common/src/main/java/azkaban/project/JdbcProjectImpl.java

```diff
@@ -814,19 +762,23 @@ public Flow fetchFlow(final Project project, final String flowId) throws Project
   }
 
   @Override
```

Comment:
This API is still useful in certain cases. Instead of deleting it, mark it @Deprecated.

Reply:
Which API are you referencing, `getProjectPermissions` or `fetchAllProjectFlows`?

Reply:
fetchAllProjectFlows

Reply:
Done - and I haven't marked it as depreciated, it works just fine.